test/e2e/config/docker.yaml

```diff
@@ -154,7 +163,16 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1.7/metadata.yaml"
-  - name: v1.8.99 # next; use manifest from source files
+  - name: "{go://sigs.k8s.io/cluster-api@v1.8}" # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
```

Comment:
Comment in line 157 is wrong (also for other providers)

Reply:
I'll drop all these comemnts. Waste of time to keep them up-to-date